docs/conf.py

```diff
@@ -30,7 +30,7 @@ def get_version(rel_path):
 author = 'Fast Machine Learning Lab'
 
 # The full version, including alpha/beta/rc tags
-release = get_version("../hls4ml/__init__.py")
+release = get_version("../hls4ml/_version.py")
```

Comment:
@vloncar do you know if this works when building the docs in the GitHub action? `_version.py` is only generated when installing hls4ml. I'm wondering if we need to `pip install -e .` in the workflow yaml.

Actually, do you know if the version has any effect on the generated docs? I thought it may be shown on the sidebar

Reply:
I was under the impression that the workflow would install the package it checked out, given that the [pre-build](https://github.com/fastmachinelearning/hls4ml/blob/main/.github/workflows/build-sphinx.yml#L17) command installs the dependencies. I could be wrong though.

Reply:
After reading a bit more, I realized the proper way would be to use `get_version()` from `setuptools_scm` itself, rather than rolling our own function for extracting version.